resources/js/components/BookingNewModal.vue

```diff
@@ -18,7 +18,7 @@
             <div class="form-group row" v-if="user.type == 'App\\Staff'">
                 <label for="client_select" class="col-md-4 col-form-label text-md-right">Select a client</label>
                 <div class="col-md-6">
-                    <select name="client_select" id="client" class="form-control" @change="onSelectClient" v-model="selectedClientId">
+                    <select name="client_select" id="client" class="form-control" @change="onSelectClient" v-model="selectedClientId" disabled="type == 'Edit'">
```

Comment:
check https://logaretm.github.io/vee-validate/ for validating forms on vuejs.I use it and it makes it a breeze

Reply:
I will address form validation in #79

Reply:
> Oh, cool. I'll check this out. I've never been happy with not having form validation and I hate that I have to pass it to the backend to check.

@beauraines you should still check it on the backend. I usually like to do data validation using [formrequest](https://laravel.com/docs/5.8/validation#form-request-validation)

Reply:
I do use formrequests for that on the back end. I was saying that it was frustrating that I have to pass it to the back end to check rather than do it on the front end. That package will probably help with that...

And you could you that view bootstrapping to pass the validation rules, so you only have to maintain them in the back end 😉

Reply:
> And you could you that view bootstrapping to pass the validation rules, so you only have to maintain them in the back end 😉

Do you have a snippet on this thing?

Reply:
No, only a concept in my head. If you keep the validation rules in the FormRequest, you can then transform them into something consumable by the frontend validator.

Reply:
But then you've to assume that the frontend will have the same validation
rules names. This might not occur everytime.